src/workspacer/Keybinds/KeybindManager.cs

```diff
@@ -56,9 +57,38 @@ public KeybindManager(IConfigContext context)
             thread.Start();
         }
 
+        public void ShowKeybindWarning(string warningMessage)
+        {
+            if (_keybindWarning == null)
+            {
+                _keybindWarning = new TextBlockMessage("workspacer keybinds", "Warning, duplicate keybinds!", warningMessage, new List<Tuple<string, Action>>()
```

Comment:
With this implementation, the warning message provided will only be used the first time and ignored all subsequent times. 

Either pre-allocate this object with a fixed message, or use the provided argument every time when invoked.